picard/ui/matchdetailsdialog/matchdetails.py

```diff
@@ -0,0 +1,211 @@
+        super().__init__(parent)
+        self.file_ = None
+        self.setWindowTitle(_("Match Details"))
+        self.columns = [
+            ('nbr',                  "Log\nSeq"),
+            ('match_dtl_entry_type', "Entry Type"),
+            ('is_new',               "Better?"),
+            ('similarity',           "Similarity"),
+            ('acoust_id',            "AcoustId ID"),
+            ('rec_id',               "Recording ID"),
+            ('rec_title',            "Recording Title"),
+            ('rec_len',              "Recording\nLength"),
+            ('rec_src_cnt',          "Recording\nSource\nCount"),
+            ('rec_artist',           "Recording\nArtist"),
+            ('rg_id',                "Release Group ID"),
+            ('rg_title',             "Release Group\nTitle"),
+            ('rg_pri_type',          "Release Group\nPri Type"),
+            ('rg_sec_type',          "Release Group\nSec Types"),
+            ('rel_id',               "Release ID"),
+            ('rel_ctry',             "Release\nCountry"),
+            ('rel_dt',               "Release\nDate"),
+            ('med_fmt',              "Medium\nFormat"),
+            ('med_pos',              "Medium\nPosition"),
+            ('med_tot_cnt',          "Medium\nCount"),
+            ('trk_id',               "Track ID"),
+            ('trk_pos',              "Track\nPosition"),
+            ('med_trk_cnt',          "Medium\nTrack\nCount"),
+            ('rel_tot_trk_cnt',      "Release\nTotal Track\nCount"),
+            ('parts',                "Scoring Parts"),
+            ('release_parts',        "Scoring Release Parts"),
+            ('recording_base_score', "Recording\nBase Score\n(Sources Ratio)")
+        ]
+
+    def load_match_details(self, track, file_):
+        """ Retrieves data and loads it to the screen/dialog box """
+        try:
+            # NOTE: some of the values, like meta, are not exposed as constants
+            fp = file_.acoustid_fingerprint if hasattr(file_, 'acoustid_fingerprint') else ""
+            dur = file_.acoustid_length if hasattr(file_, 'acoustid_length') else ""
+
+            fingerprint_lkp_url = ("https://" + ACOUSTID_HOST + "/v2/lookup"
+                                   + "?meta=recordings%20releasegroups%20releases%20tracks%20compress%20sources"
+                                   + "&fingerprint=" + fp
+                                   + "&duration=" + str(dur)
+                                   + "&client=" + ACOUSTID_KEY
+                                   + "&clientversion=" + PICARD_VERSION_STR
+                                   + "&format=json") if fp is not None else ""
+
+            json = match_details_log.get_file_result_json(file_.filename) if (file_ is not None) else ""
+            if json is not None:
+                self.set_json_edit(json)
+            if file_ is not None:
```

Comment:
Test already done few lines above, may be code can be reordered to do it once only.